<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="toolTip">
       <string>Hello, world</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <widget class="QToolButton" name="toolButton">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>30</y>
             <width>21</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_2">
           <property name="geometry">
            <rect>
             <x>210</x>
             <y>70</y>
             <width>21</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
          <widget class="QToolButton" name="toolButton_3">
           <property name="geometry">
            <rect>
             <x>220</x>
             <y>110</y>
             <width>21</width>
             <height>18</height>
            </rect>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>121</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>180</y>
             <width>101</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_4">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>180</y>
             <width>101</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_3">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>80</y>
             <width>121</width>
             <height>61</height>
            </rect>
           </property>
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
          <widget class="QGroupBox" name="groupBox">
           <property name="geometry">
            <rect>
             <x>300</x>
             <y>60</y>
             <width>180</width>
             <height>141</height>
            </rect>
           </property>
           <property name="title">
            <string>GroupBox</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QProgressBar" name="progressBar">
                <property name="value">
                 <number>24</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Hello, world</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="verticalSlider">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QSlider" name="horizontalSlider">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QComboBox" name="comboBox">
           <property name="geometry">
            <rect>
             <x>370</x>
             <y>10</y>
             <width>72</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>220</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>250</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>RadioButton</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>280</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>310</y>
             <width>161</width>
             <height>19</height>
            </rect>
           </property>
           <property name="text">
            <string>CheckBox</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScrollBar" name="verticalScrollBar">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QScrollBar" name="horizontalScrollBar">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>20</y>
          <width>21</width>
          <height>131</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>130</y>
          <width>160</width>
          <height>19</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>24</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>286</width>
          <height>294</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTabWidget" name="tabWidget_2">
           <widget class="QWidget" name="tab_3">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_4">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTabWidget" name="tabWidget_5">
           <property name="tabPosition">
            <enum>QTabWidget::East</enum>
           </property>
           <widget class="QWidget" name="tab_9">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_10">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QTabWidget" name="tabWidget_3">
           <property name="tabPosition">
            <enum>QTabWidget::West</enum>
           </property>
           <widget class="QWidget" name="tab_5">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_6">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QTabWidget" name="tabWidget_4">
           <property name="tabPosition">
            <enum>QTabWidget::South</enum>
           </property>
           <widget class="QWidget" name="tab_7">
            <attribute name="title">
             <string>Tab 1</string>
            </attribute>
           </widget>
           <widget class="QWidget" name="tab_8">
            <attribute name="title">
             <string>Tab 2</string>
            </attribute>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>ページ</string>
       </attribute>
       <widget class="QListWidget" name="listWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>20</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>230</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>ページ</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>PushButton</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>ページ</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_file_open"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="actionCu_t"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
   </widget>
   <widget class="QMenu" name="menuHoge">
    <property name="title">
     <string>Hoge</string>
    </property>
    <addaction name="action_fuga"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuHoge"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_file_open"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_file_open">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New...</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="actionCu_t">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_fuga">
   <property name="text">
    <string>Fuga</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>progressBar_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
